### PR TITLE
Create sharath.txt

### PR DESCRIPTION
**Pull request by sharath29**
This Pull Request resolves Issue #1 #9 

Description:
In this Pull Request we have done the following additions/edits for the code:

This solves #1 for Participants details.

### CODE REVIEW

File: sting_reverse/string_reverse.c [hunk 1]
```diff
@@ -0,0 +1,18 @@
+This Pull Request resolves Issue #9
```
Comment (salman-bhai) [inline sting_reverse/string_reverse.c]: This shoule be a comment using the symbol `//`

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,8 +1,5 @@
-This Pull Request resolves Issue #<IssueNumber>
+This Pull Request resolves Issue #9
```
Comment (salman-bhai) [inline PULL_REQUEST_TEMPLATE.md]: Do not tamper with this file!

Comment (pvgupta24) [inline PULL_REQUEST_TEMPLATE.md]: You have referenced the issue in PR description. This file was just a template to describe what you need to write in PR description. 
Do not make changes in this file.

**Comment by salman-bhai** [comment]
Are there any updates here or should we close the PR and label it as `spam`?

**Comment by sharath29** [comment]
I am busy for a while I will update it soon. Sorry for the delay.

**Comment by salman-bhai** [comment]
Please do it soon! We'd like to close our PRs

**Comment by salman-bhai** [comment]
Please change the file as it was earlier. You are not supposed to edit that file!

**Pull request closed without merging**